packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/pin/usePinDiagramElements.tsx

```diff
@@ -74,9 +74,11 @@ export const usePinDiagramElements = (): UsePinDiagramElements => {
         elementIds: nodeId,
         pinned,
       };
-      pinElementMutation({ variables: { input } });
+      if (!readOnly) {
```

Comment:
It should not be possible to trigger this in readonly

Reply:
Done